### PR TITLE
T15- Isogram

### PR DESCRIPTION
**Pull request by Priyanka15-12**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");
```
Comment (MetaNaveen) [inline Program.cs:9]: isogram variable should have been used in here.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+      }
+
+      static bool Isogram (string iso) {
+         iso = iso.ToLower ();
+         HashSet<char> chars = new HashSet<char> ();
+         foreach (char c in iso) {
```
Comment (MetaNaveen) [inline Program.cs:15]: You could avoid the curly brace here. Can you think of how to do?

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");
+      }
+
+      static bool Isogram (string iso) {
+         iso = iso.ToLower ();
+         HashSet<char> chars = new HashSet<char> ();
```
Comment (MetaNaveen) [inline Program.cs:14]: Can be initialized with shorthand form **new ()**.

File: Program.cs [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");
+      }
+
+      static bool Isogram (string iso) {
+         iso = iso.ToLower ();
```
Comment (MetaNaveen) [inline Program.cs:13]: The lower case conversion could be done in the loop itself.

**Review by MetaNaveen — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (Priyanka15-12) [reply Program.cs:8]: Removed isogram variable line.

File: Program.cs (hunk 2 above)
Comment (Priyanka15-12) [reply Program.cs]: Removed curly braces

File: Program.cs (hunk 3 above)
Comment (Priyanka15-12) [reply Program.cs]: Shorthand form applied

File: Program.cs (hunk 4 above)
Comment (Priyanka15-12) [reply Program.cs]: Lower case conversion done in loop

**Comment by MetaNaveen** [comment]
Working fine.

**Pull request closed without merging**